Classes/Public/Manager/Submanagers/OCTSubmanagerCallsDelegate.h

```diff
@@ -0,0 +1,34 @@
+//
+//  OCTSubmanagerCallsDelegate.h
+//  objcTox
+//
+//  Created by Chuong Vu on 6/23/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+@class OCTCall;
+@class OCTSubmanagerCalls;
+
+@protocol OCTSubmanagerCallDelegate <NSObject>
+
+/**
+ * Delegate for when we receive a call.
+ **/
+- (void)callSubmanager:(OCTSubmanagerCalls *)callSubmanager receiveCall:(OCTCall *)call audioEnabled:(BOOL)audioEnabled videoEnabled:(BOOL)videoEnabled;
+
+/**
+ * Audio bitrate has changed
+ **/
+- (void)callSubmanager:(OCTSubmanagerCalls *)callSubmanager audioBitRateChanged:(OCTToxAVAudioBitRate)bitRate stable:(BOOL)stable forCall:(OCTCall *)call;
+
+/**
+ * Video bitrate has changed
+ **/
+- (void)callSubmanager:(OCTSubmanagerCalls *)callSubmanager videoBitRateChanged:(OCTToxAVAudioBitRate)bitRate stable:(BOOL)stable forCall:(OCTCall *)call;
+
+/**
+ * Call state has changed
+ **/
+- (void)callSubmanager:(OCTSubmanagerCalls *)callSubmanager stateChanged:(OCTToxAVCallState)state forCall:(OCTCall *)call;
```

Comment:
You are right, we have RBQFetchedResultsController for that. I didn't think about BOOL properties when I was writing that. With properties there is no need for this method at all. So you don't need a delegate method for OCTCall change. :-)

BTW, there is a hack to monitor particular object with RBQ - do fetch with predicate `uniqueIdentifier == %@", call.uniqueIdentifier`.